src/lf_apply.rs

```diff
@@ -426,15 +426,17 @@ unsafe fn filter_plane_cols_uv<BD: BitDepth>(
     // filter edges between columns (e.g. block1 | block2)
     let lf_sb = &f.dsp.lf.loop_filter_sb;
     let len = endy4 - starty4;
+    let u_dst = |x| u_dst + x * 4;
+    let v_dst = |x| v_dst + x * 4;
+    u_dst(w).as_ptr::<BD>(); // Bounds check
+    v_dst(w).as_ptr::<BD>(); // Bounds check
     let mask = &mask[..w];
     let lvl = &lvl[..w];
     for x in 0..w {
         if !have_left && x == 0 {
             continue;
         }
-        let u_dst = u_dst + x * 4;
-        let v_dst = v_dst + x * 4;
-        let mask = &mask[x];
+        let mask = &mask[x % mask.len()]; // To elide the bounds check.;
```

Comment:
Same thing here, we're already doing `let mask = &mask[..w];` so either this is redundant or that is.